pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con-about-document-attributes.adoc

```diff
@@ -0,0 +1,57 @@
+[id="con-about-document-attributes_{context}"]
+= About document attributes
```

Comment:
I suggest modifying the title. I'd say this module explains how to set the attribute file for your docs that are published in PVB2/Jupiter. The title "About document attributes" sounds like I'll learn about using attributes

Reply:
How about "About using document attributes in Pantheon"?

Reply:
Yes, I think that would be a nice fit